apps/cms/scripts/seed.mjs

```diff
@@ -98,6 +98,16 @@ async function main(args) {
    */
   console.log('Seeding database...')
 
+  try {
+    await createEntityRecords(
+      'languages',
+      [{ code: 'en-US', name: 'English' }],
+      token
+    )
+  } catch (err) {
+    console.log('Language already exists.')
+  }
```

Comment:
When using the new Directus schema snapshots, we won't get the language codes by default. So this addition to the seed script will ensure `en-US` language is present.